feat(content): The Successors - Enter the Frogtopus Aristocracy (#10296)
endless-sky/endless-sky: data/successors/successor hails.txt

# Copyright (c) 2024 by Daeridanii
#
# Endless Sky is free software: you can redistribute it and/or modify it under the
# terms of the GNU General Public License as published by the Free Software
# Foundation, either version 3 of the License, or (at your option) any later version.
#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A

> # PARTICULAR PURPOSE.  See the GNU General Public License for more details.

Reviewer: ```suggestion
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
```

#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

### SUCCESSOR CIVILIAN

phrase "successor friendly hail" # Wary hails
	word
		"Art thou the 'noble visitor' of whom the Houses speak?"
		"Art thou the 'honored guest' the Houses do parade?"
		"Art thou the fiendish creature that hails from distant lands?"
		"Art thou the strange-ish traveler that hails from distant stars?"
		"Art thou the alien visitor within our noble lands?"
		"Art thou the creature've history returned to seek the present?"
		"Thy ship and countenance seem strange-ish and bizarre,"
	word
		" / "
	word
		"I note thy presence but remain thee wary."
		"If corrupted be thy soul, then prithee keep thy distance."
		"If thou art friend to me, so I shall be to thee."
		"What is thy goal within these stars? To wander in delight?"
		"To whom and what thou does thou lend thy own strength?"
		"What is thy goal within these stars? To observe or interfere?"
		"What is thy goal within these stars? Be a trader or a traitor?"

> 

Reviewer 2: ```suggestion

```

phrase "successor friendly hail" # Brag hails 1
	word
		"I never fly without"
	word
		" "
	word
		"a trusty scanner"
		"my trusty scanner"
		"a larger fuel tank"
		"communications"
		"some conversation"
		"a stock of crab-cubes"
		"a stock of kelp-packs"
		"a week's supply-cache"
	word
		", / "
	word

> 		"I suggest you do the same if you have interest."

Reviewer 2: ```suggestion
		"I suggest you do the same, if you have interest."
```

		"It makes the many hours pass by quickly."
		"The dark of space is lightened by its presence."
		"There's little point in taking off without it."

phrase "successor friendly hail" # Job hails
	word
		"My "
	phrase
		"relations"
	word
		" always told me: be a "

> 		" has just earned the title: "

Reviewer 2: It's plural above, unless that wasn't supposed to work that way.
```suggestion
		" have just earned the title: "
```

		" used to know a friendly "
		" helped me find a friendly "
		" just got married to a "
		" told me: never trust a "
		" used to be a skillful "
	phrase
		"occupation"
	word
		" / "
	word
		"With these long days, I wish that they were me."
		"A noble job, and yet not one for me."
		"It's not my place to give critique, and yet..."
		"I hope that I will see them soon again."
		"Hast thou met them? Do they carry any news?"

phrase "successor friendly hail" # Conspiracy hails
	word
		"I hope that thou will keep this thought between us:"
		"A friend of mine had told me once in secret:"

> 		"I have heard from good reputed, honest lips that"

Reviewer 2: ```suggestion
		"I have heard from well reputed, honest lips that:"
```

> 		"The High Houses try to hide it, but I know that"

Reviewer 2: ```suggestion
		"The High Houses try to hide it, but I know that:"
```

		"I've learned in many years of honest pil'ting:"
		"If thou art willing to believe me:"
		"Doubt my words if knave'ry is thy goal, but:"
	word
		" / "
	word
		"It's the Drak who caused the end of the old empire."
		"It's the Quarg who caused the end of the old empire."
		"It's the Hai who caused the end of the old empire."
		"It's the Korath caused the end of the old empire."
		"The High Houses by themselves destroyed the empire."
		"The best trade routes are ones not prone to scanning." 5
		"The best trade routes are those that make thy money." 5
		"The best trade routes are hidden from thy sensors." 2
		"There are nightmares in the void the Houses fear."
		"The empty systems here are prone to strangeness."

> 

Reviewer 2: ```suggestion

```

phrase "successor hostile hail"
	word
		"Thou hast betrayed our trust and for what purpose!?"
		"Our foolishness it was, we see, to trust thee;"
		"Cruel, malignant, violent, petty creature."
		"Vile! Thy deception is exposed now."
		"Your death will leave no absence on the cosmos."
		"Our friendship now, you find, has been retracted."
	word
		" / "
	word
		"So sorry thou shalt meet a violent end."
		"The Houses will unburden thee of life."
		"Prepare to shuffle off thy mortal coil."
		"The gentle hand of Death close up thy eye."
		"No satisfaction lives in our reprisal."

> 

Reviewer 2: ```suggestion

```

phrase "successor friendly disabled hail"
	word

> 		"Thy prompt assist t'would honor thee,"

Reviewer 2: ```suggestion
		"Thy prompt assistance t'would honor thee,"
```

		"Alas, our vessel's lay'en low,"

> 		"'Help!' we ask, fair visitor,"

Reviewer 2: ```suggestion
		`"Help!" we ask, fair visitor,`
```

		"Our valiant craft's been stricken, friend,"
	word
		" / "
	word
		"Make haste, we thee implore."
		"Lend aid if 'tis in reach."
		"Thy help would make us whole."
		"'Tis noble to be kind and wise to help."

> 

Reviewer 2: ```suggestion

```

phrase "successor hostile disabled hail"
	word
		"Ruinous creature, see now thy bitter handiwork!"
		"Thy deceit's laid low our fairest vessel!"
		"Bitter knave, thoughtless in kind and action!"
		"Cruel deceiver, we observe our fault 'twas trusting thee,"
	word
		" / "
	word
		"The Houses shall bring thy bitter self to justice!"
		"Spare us but our lives if not our ship!"
		"Thou art cursed, foolish, cruel, and sure to die."
		"Our families shall avenge us, no matter how!"

> ### HIGH HOUSES: GENERIC

Reviewer 2: ```suggestion
# High Houses: Generic
```

phrase "high houses friendly hail"
	word
		"We welcome thee to join us in our space"
		"Thy presence in our lovely space is welcome"
		"We note thy visitation and accept"
		"'Noble friend', thou art by our decree"
		"'Honored guest,' within our skies thou art"
		"'Regal sojourner,' thy name shall be with us"
	word
		" / "
	word
		"So that thou might enjoy our pleasant worlds."
		"So that thou might enjoy our verdant worlds."
		"So that thou might make note of our advance."
		"So that thou might find joy in our success."
		"So that thou may bear witness to our joy."

> 

Reviewer 2: ```suggestion

```

phrase "high houses friendly hail"
	word
		"By noble lineage do we exalt us all"
		"With joyful strength and honest might we fly"
		"No evil shall impede our noble goal"
		"Our goal's a true and simple thing:"
		"With watching eyes and gallant hearts"
	word
		" / "
	word
		"To build for all a great and lasting peace."
		"To make for all a true and joyful peace."
		"To keep us all secure and bright in peace."
		"To promote the joy and welfare of us all."

> 

Reviewer 2: ```suggestion

```

phrase "high houses hostile hail"
	word
		"Thy deception marks the apex of thy ill,"
		"Thy cowardice is the basis of thy fame,"
		"Our foolishness it was, we see, to trust thee;"
		"A knave and coward thou hast clearly proved thyself;"
		"Cowards die many times before their deaths,"
	word
		" / "
	word
		"Our duty shall relieve thee of thy life."
		"May thou in death find peace thou lacked in life."
		"May thy death be valorous and quick."
		"May, in death, thou recognize thy wrongs."
		"Enlightened now, we are to thy malignity."
		"May thy knavish and deceptive heart now cease."
		"The valiant never taste of death but once."

> 

Reviewer 2: ```suggestion

```

phrase "high houses hostile hail"
	word
		"We take no pleasure in this graven duty,"
		"Thy venom shall be met with our resolve:"
		"Our graven duty 'tis to end thy cruelty:"
	word
		" / "
	word
		"We'll make thy death a quick and painless thing."
		"Our thorns shall sprout in thy rebellious flesh."
		"May thy death be valorous and quick."
		"May thy pain be consuming and quick."

> 

Reviewer 2: ```suggestion

```

> phrase "high houses hostile disabled hail" 2

Reviewer 2: Can you do this? Or did you mean it this way:
```suggestion
phrase "high houses hostile disabled hail 2"
```

	word
		"Ruinous cur, our vengeance shall yet find thee!"
		"Revenge is a dish best served cold,"
		"Thou might break our ship but shall never break our spirit!"

> 		"Putrefy and fester, knave deciever!"

Reviewer 2: ```suggestion
		"Putrefy and fester, knave deceiver!"
```

		"Board us and we shall fight to the last!"
	word
		" / "
	word
		"Our friends shall avenge thy bitter cruelty!"
		"Let thy rotten flesh be consumed in flame!"
		"Our friends shall yet let thee know fear."
		"Thy only noble act shall be to perish."

> 

Reviewer 2: ```suggestion

```

phrase "high houses hostile disabled hail"
	word
		"From void's heart, we stab at thee! / For the good of hate, we spit our last breaths at thee!"
		"Cowards die many times before their deaths, / But the valiant never taste of death but once!"

> ### HIGH HOUSES: OLD HOUSES

Reviewer 2: ```suggestion
# High Houses: Old Houses
```

phrase "old houses friendly hail" # Boast hails
	word
		"The"
	word
		" arm of "
		" spear of "
		" sword of "
		" shield of "
		" prime of "
		" best of "
	word
		"the Old Houses"
	word
		" in"
		" with"
	word
		" joy"
		" pride"
		" strength"
		" glory"
	word
		" we are;"
	word
		" / "
	word
		"A simple truth in which we take our pride."
		"An honest fact 'tis plain for all to see."
		"A simple truth in which we take our joy."
		"A truth that we wear on our sleeves."
		"A position that we hold with utmost glee."
		"An honor that we never shall deny."
		"A joy in word and action, purely said."

> 

Reviewer 2: ```suggestion

```

phrase "old houses friendly hail"
	phrase
		"high houses friendly hail"

> 

Reviewer 2: ```suggestion

```

phrase "chydiyi friendly hail"
	phrase
		"old houses friendly hail"

> 

Reviewer 2: ```suggestion

```

phrase "myurej friendly hail"
	phrase
		"old houses friendly hail"

> 

Reviewer 2: ```suggestion

```

phrase "sioeora friendly hail"
	phrase
		"old houses friendly hail"

> ### HIGH HOUSES: NEW HOUSES

Reviewer 2: ```suggestion
# High Houses: New Houses
```

phrase "new houses friendly hail" # Boast hails
	word
		"The"
	word
		" arm of "
		" spear of "
		" sword of "
		" shield of "
		" prime of "
		" best of "
	word
		"the New Houses"
	word
		" in joy"
		" with pride"
		" with joy"
	word
		" we are;"
	word
		" / "
	word
		"A simple truth in which we take our pride."
		"An honest fact 'tis plain for all to see."
		"A simple truth in which we take our joy."

> 

Reviewer 2: ```suggestion

```

phrase "new houses friendly hail" 2
	phrase
		"high houses friendly hail"

> 

Reviewer 2: ```suggestion

```

phrase "kaatrij friendly hail"
	word

> 		`On gossamer wings with fang'ed claws / our noble house flies unhindered.`

Reviewer 2: ```suggestion
		"On gossamer wings with fang'ed claws / our noble house flies unhindered."
```

> 		`In our diversity we are united / and in our unity we find our strength.`

Reviewer 2: ```suggestion
		"In our diversity we are united / and in our unity we find our strength."
```

> 		`Remember the fallen.`

Reviewer 2: ```suggestion
		"Remember the fallen."
```

> 		`We endure, together.`

Reviewer 2: ```suggestion
		"We endure, together."
```

> 		`Now and always.`

Reviewer 2: ```suggestion
		"Now and always."
```

> 

Reviewer 2: ```suggestion

```

phrase "kaatrij friendly hail"
	phrase
		"new houses friendly hail"

> 

Reviewer 2: ```suggestion

```

phrase "seineq friendly hail"
	word

> 		`For ten thousand years / for ten thousand more!`

Reviewer 2: ```suggestion
		"For ten thousand years / for ten thousand more!"
```

> 		`Our alliance shall endure.`

Reviewer 2: ```suggestion
		"Our alliance shall endure."

```

phrase "seineq friendly hail"
	phrase
		"new houses friendly hail"

> 

Reviewer 2: ```suggestion

```

phrase "aqrabe friendly hail"
	phrase
		"new houses friendly hail"

> ### PEOPLE'S HOUSES

Reviewer 2: ```suggestion
# People's Houses
```

phrase "people's houses friendly hail"
	word
		"Revel in thy freedom and endurance."

> 		"I intend to thee no harm nor malefeasance."

Reviewer 2: ```suggestion
		"I intend to thee no harm nor malfeasance."
```

		"Art thou the alien visitor within our noble lands?"
		"Art thou the creature've history returned to seek the present?"
		"Thy ship and countenance seem strange-ish and bizarre,"
	word
		" / "
	word
		"I note thy presence but remain thee wary."
		"If corrupted be thy soul, then prithee keep thy distance."
		"If thou art friend to me, so I shall be to thee."
		"What is thy goal within these stars? To wander in delight?"
		"To whom and what thou does thou lend thy own strength?"
		"What is thy goal within these stars? To observe or interfere?"
		"What is thy goal within these stars? Be a trader or a traitor?"

phrase "successor friendly hail" # Brag hails 1
	word
		"I never fly without"
	word
		" "
	word
		"a trusty scanner"
		"my trusty scanner"
		"a larger fuel tank"
		"communications"
		"some conversation"
		"a stock of crab-cubes"
		"a stock of kelp-packs"
		"a week's supply-cache"
	word
		", / "
	word
		"I suggest you do the same if you have interest."
		"It makes the many hours pass by quickly."
		"The dark of space is lightened by its presence."
		"There's little point in taking off without it."

phrase "successor friendly hail" # Job hails
	word
		"My "
	phrase

> 		"relations"

Reviewer 3: Where is the `relations` phrase defined? I can't seem to find it anywhere.

Reviewer 2: It is defined in `successors.txt`.

		"Prepare to shuffle off thy mortal coil."
		"The gentle hand of Death close up thy eye."
		"No satisfaction lives in our reprisal."

phrase "successor friendly disabled hail"
	word
		"A prompt assist would honor thee,"
		"Alas, our vessel's lay'en low,"
		`"Help!" we ask, fair visitor,`
		"Our valiant craft's been stricken, friend,"
	word
		" / "
	word
		"Make haste, we thee implore."
		"Lend aid if 'tis in reach."
		"Thy help would make us whole."
		"'Tis noble to be kind and wise to help."

phrase "successor hostile disabled hail"
	word
		"Ruinous creature, see now thy bitter handiwork!"
		"Thy deceit's laid low our fairest vessel!"
		"Bitter knave, thoughtless in kind and action!"
		"Cruel deceiver, we observe our fault 'twas trusting thee,"
	word
		" / "
	word
		"The Houses shall bring thy bitter self to justice!"
		"Spare us but our lives if not our ship!"
		"Thou art cursed, foolish, cruel, and sure to die."
		"Our families shall avenge us, no matter how!"

# High Houses: Generic

phrase "high houses friendly hail"
	word
		"We welcome thee to join us in our space"
		"Thy presence in our lovely space is welcome"
		"We note thy visitation and accept"

> 		"'Noble friend', thou art by our decree"
> 		"'Honored guest,' within our skies thou art"
> 		"'Regal sojourner,' thy name shall be with us"

Reviewer 3: ```suggestion
		`"Noble friend," thou art by our decree`
		`"Honored guest," within our skies thou art`
		`"Regal sojourner," thy name shall be with us`
```

American English

		"Our alliance shall endure."

phrase "seineq friendly hail"
	phrase
		"new houses friendly hail"

phrase "aqrabe friendly hail"
	phrase
		"new houses friendly hail"

# People's Houses

phrase "people's houses friendly hail"
	word
		"Revel in thy freedom and endurance."
		"I intend to thee no harm nor malfeasance."
		"We fly the flag of freedom from oppression."
		"The People's Houses offer true equality."
	word
		" / "
	word
		"Live without the chains of aristocracy."
		"Remain unshackled from the listless norm."
		"The People's Houses are ready to assist."

phrase "people's houses hostile hail"
	word
		"Art thou a knave in title same as action!?"
		"Thou art nothing but a foolish slave to power!"
		"Art thou blind and deaf to suff'ring or just evil?"
		"If thou wouldst drop thy cargo, we'll be going."
		"No harm shall come to those who drop their cargo."
		"Thou art knavish in thy friendships and thy action."
	word
		" / "
	word
		"Thy blood shall grace the inside of thy starship."
		"Resistance shall be met with execution."
		"I hope thy pain is great and everlasting."
		"Allow me to unburden thee of mortality."
		"Prepare to shuffle off thy mortal coil."

> 		"The sudden hand of Death close up thy eye."

Reviewer 3: ```suggestion
		"The sudden hand of Death close up thine eye."
```

phrase "successor friendly hail" # Job hails
	word
		"My "
	phrase
		"successor relation"
	word
		" always told me: be a "
		" has just earned the title: "
		" used to know a friendly "
		" helped me find a friendly "
		" just got married to a "
		" told me: never trust a "
		" used to be a skillful "
	phrase
		"successor occupation"
	word
		" / "
	word
		"With these long days, I wish that they were me."
		"A noble job, and yet not one for me."
		"It's not my place to give critique, and yet..."
		"I hope that I will see them soon again."
		"Hast thou met them? Do they carry any news?"

phrase "successor friendly hail" # Conspiracy hails
	word
		"I hope that thou will keep this thought between us:"
		"A friend of mine had told me once in secret:"
		"I have heard from well reputed, honest lips that:"
		"The High Houses try to hide it, but I know that:"
		"I've learned in many years of honest pil'ting:"
		"If thou art willing to believe me:"
		"Doubt my words if knave'ry is thy goal, but:"
	word
		" / "
	word
		"It's the Drak who caused the end of the old empire."
		"It's the Quarg who caused the end of the old empire."
		"It's the Hai who caused the end of the old empire."

> 		"It's the Korath who caused the end of the old empire."

Reviewer 4: Which species are the Successors aware of? They live quite far from the Hai, I believe.

Author: The Hai _are_ pretty distant so it makes sense for the hails to be blaming the Coalition or another more proximate species instead.
Officially, the Successors's historical experimentation with jump drives makes it plausible for them to be aware of all the spacefaring species that were around ca. 0 -- 1500 CE. Of these, it's likely the Quarg, Drak, Ka'het, Coalition, and Korath that they are most familiar with (roughly in order).

		"The High Houses by themselves destroyed our empire."
		"The best trade routes are ones not prone to scanning." 5
		"The best trade routes are those that make thee money." 5
		"The best trade routes are hidden from thy sensors." 2
		"There are nightmares in the void that the Houses fear."
		"The empty systems here are prone to strangeness."

phrase "successor hostile hail"
	word
		"Thou hast betrayed our trust and for what purpose!?"
		"Our foolishness it was, we see, to trust thee;"
		"Cruel, malignant, violent, petty creature."
		"Vile! Thy deception is exposed now."
		"Your death will leave no absence on the cosmos."
		"Our friendship now, you find, has been retracted."
	word
		" / "
	word
		"So sorry thou shalt meet a violent end."
		"The Houses will unburden thee of life."
		"Prepare to shuffle off thy mortal coil."
		"The gentle hand of Death close up thy eye."
		"No satisfaction lives in our reprisal."

phrase "successor friendly disabled hail"
	word

> 		"A prompt assist would honor thee,"

Author: "Assist" as a noun has some precedent, though it is primarily used in American English and is less prevalent in other dialects. I think in context it shouldn't be too confusing regardless, and "assistment" is a lot clunkier, in my opinion.

		"Alas, our vessel's lay'en low,"
		`"Help!" we ask, fair visitor,`
		"Our valiant craft's been stricken, friend,"
	word
		" / "
	word
		"Make haste, we thee implore."
		"Lend aid if 'tis in reach."
		"Thy help would make us whole."
		"'Tis noble to be kind and wise to help."

phrase "successor hostile disabled hail"
	word
		"Ruinous creature, see now thy bitter handiwork!"
		"Thy deceit's laid low our fairest vessel!"
		"Bitter knave, thoughtless in kind and action!"
		"Cruel deceiver, we observe our fault 'twas trusting thee,"
	word
		" / "
	word
		"The Houses shall bring thy bitter self to justice!"
		"Spare us but our lives if not our ship!"
		"Thou art cursed, foolish, cruel, and sure to die."
		"Our families shall avenge us, no matter how!"

# High Houses: Generic

phrase "high houses friendly hail"
	word
		"We welcome thee to join us in our space"
		"Thy presence in our lovely space is welcome"
		"We note thy visitation and accept"
		`"Noble friend," thou art by our decree`
		`"Honored guest," within our skies thou art`
		`"Regal sojourner," thy name shall be with us`
	word
		" / "
	word

> 		"So that thou might enjoy our pleasant worlds."

Author: The syllable it provides is important for the sentences' meter, so I prefer to keep it unless there's a strong reason it should go/be changed.

		"Woe, destruction, ruin and decay; / The worst is death, and death will have its day." # Richard II, Act 3, scene 2
		"Cur, keep back, or else embrace thy death; / Come not within the measure of my wrath." # The Two Gentlemen of Verona, Act 5, scene 4

# High Houses: Old Houses

phrase "old houses friendly hail" # Boast hails
	word
		"The"
	word
		" arm of "
		" spear of "
		" sword of "
		" shield of "
		" prime of "
		" best of "
	word
		"the Old Houses"
	word
		" in"
		" with"
	word
		" joy"
		" pride"
		" strength"
		" glory"
	word
		" we are;"
	word
		" / "
	word
		"A simple truth in which we take our pride."
		"An honest fact 'tis plain for all to see."
		"A simple truth in which we take our joy."
		"A truth that we wear on our sleeves."
		"A position that we hold with utmost glee."
		"An honor that we never shall deny."
		"A joy in word and action, purely said."

phrase "old houses friendly hail"
	phrase
		"high houses friendly hail"

phrase "chydiyi friendly hail"
	phrase
		"old houses friendly hail"

phrase "myurej friendly hail"
	phrase
		"old houses friendly hail"

> phrase "sioeora friendly hail"

Author: All of the Houses pull from the appropriate "old houses/new houses hails" list, which themselves pull from the general "high houses hails" list. _Some_ of the Houses also have unique hails just for them, which are mixed in with the other appropriate categories.

This means there are hails shared by all of the Houses, hails that are shared only by Old/New Houses, and some hails exclusive to a single House. Not all of the Houses have that third category of hails right now. The data format means we have the capacity to add those for each of the Houses, and I no doubt will do so in future PRs (and others will of course be welcome to do the same!), but adding House-specific hails for every House is not a priority for me at this moment.

	word
		" / "
	word
		"T'was the scheming Drak who brought ruin to our empire."

> 		"The treach'rous Quarg's who caused the end of the old empire."

Reviewer 5: Honestly, a mispluralization of "Quarg" is fun...

> 		"The treach'rous Quarg's who caused the end of the old empire."

Author: "Quarg're" is grammatically correct, but I don't like the way it sounds, so I think I'll leave this as "Quarg's" for the moment.
